src/components/Header/menu-links.jsx

```diff
@@ -7,9 +7,12 @@ const MenuLinks = () => {
     const { has_wallet_account } = useSelector(state => state.client);
     return (
         <div className='header__menu-item header__menu-links client_logged_in'>
-            {!has_wallet_account && config.reports.visible && (
+            {config.reports.visible && (
```

Comment:
```suggestion
            {config.tradershub.visible && (
```

@maryia-matskevich-deriv I think we are still referring to config.reports.visible 
It should be config.tradershub.visible

can you please check again?

Reply:
we have this logic on the master branch, I didn't change this part of the logic cos we haven't for this change requirements
<img width="1312" alt="Screenshot 2024-02-21 at 09 36 07" src="https://github.com/deriv-com/binary-bot/assets/103181650/a435c91f-9ed9-4ebf-a907-c5c49794a267">

Reply:
we need the non-wallet account to redirect to the trader hub, for the wallet account to redirect to the page wallets, the remaining part of the logic is the same

Reply:
The logic in master branch is incorrect as I mentioned here: [https://github.com/deriv-com/binary-bot/pull/397#discussion_r1495235416](https://github.com/deriv-com/binary-bot/pull/397#discussion_r1495235416)
@maryia-matskevich-deriv 
Let's leave it if it is time consuming, I will fix it later